test/e2e/scheduling/ubernetes_lite.go

```diff
@@ -62,7 +62,7 @@ var _ = SIGDescribe("Multi-AZ Clusters", func() {
 
 // SpreadServiceOrFail check that the pods comprising a service
 // get spread evenly across available zones
-func SpreadServiceOrFail(f *framework.Framework, replicaCount int, image string) {
+func SpreadServiceOrFail(f *framework.Framework, replicaCount int) {
```

Comment:
Indeed, I'll change this back, since it's outside the scope of this PR, even if the arg is unused.

I think I was going to remove the "image" variable entirely from this file, since it seemed to be unused, but apparently there's still a test using it.